docs/root/configuration/http/http_filters/codec_filter.rst

```diff
@@ -0,0 +1,8 @@
+.. _config_http_filters_codec:
+
+Codec
+======
+
+The codec filter is the only supported terminal filter for upstream filters.
+
+There currently are no supported configuration options for the codec filter.
```

Comment:
Nit: If we are going to publicly expose this, can we make it more clear what this is with the name? "upstream_codec" or possible even "upstream_terminal_filter" or something?